src/features/surveys/store.ts

```diff
@@ -87,6 +88,21 @@ const surveysSlice = createSlice({
       item.isLoading = false;
       item.loaded = new Date().toISOString();
     },
+    surveySubmissionsLoad: (state, action: PayloadAction<number>) => {
+      // TODO: Segregate submissions by survey ID
+      // I just wrote this for preventing never used error for commit
+      const id = action.payload;
+      state.submissionList.error = id;
```

Comment:
This could cause a bug. Please do something else, e.g. add a eslint comment to disable the linting error.